### ISSUE TITLE
protonmail bridge package

### ISSUE THREAD

**Issue by podiki** [aartaka/guix-config#1]
Hello!

I wondered if anyone had managed to make some protonmail bridge package for Guix and found yours. Has it worked for you? I was just trying to run/update it, and it seems it uses an incompatible version of the QT libraries. The cli version seems to work better, though haven't tried doing more than starting it up so far.

**Comment by aartaka** [comment]
No, I haven't gotten further than CLI either :(

Does it connect to proton servers successfully in your case? In my case my (former, I'm not in Russia anymore) shady ISP was blocking any requests to Proton, so I was unable to test :D

**Comment by podiki** [comment]
Yes, as far as I can tell the CLI works fine, just set it up. Looks like QT6 will be coming soon for the bridge: https://github.com/ProtonMail/proton-bridge/issues/211#issuecomment-1229542576 Not sure if that will make it easier or not. I've also looked into peroxide and almost have it building, while someone has submitted to guix hydroxide (but that didn't give me full IMAP access when I tried).

**Comment by podiki** [comment]
Oh. Meant to ask if you mind if I submit a modified version of your package to Nonguix? It'll be a binary and will only have the CLI version functional, but would be better than nothing. Let me know your preferred copyright line so I can add it.

**Comment by aartaka** [comment]
> Oh. Meant to ask if you mind if I submit a modified version of your package to Nonguix? It'll be a binary and will only have the CLI version functional, but would be better than nothing. Let me know your preferred copyright line so I can add it.

I don't mind! The copyright line is:
```
Artyom Bologov <aartaka@protonmail.com>
```

**Comment by podiki** [comment]
Thanks! I'll see about submitting that, though I also did get peroxide working I think. That will take some time to polish for upstream Guix (including writing a service) but would be a good long term solution.

I'll update you here when anything happens, thanks again for a nice starting point!
